modules/trusted-firmware-m/tfm_boards/CMakeLists.txt

```diff
@@ -115,7 +115,7 @@ if (${TFM_PARTITION_CRYPTO})
     tfm_sprt
   )
 
-  if (${TFM_PARTITION_INITIAL_ATTESTATION})
+  if ((${TFM_PARTITION_INITIAL_ATTESTATION}) AND CONFIG_IDENTITY_KEY)
```

Comment:
This will not work for other nRF SoCs.
Would it be better to do "if 5340 or 9160 or etc etc"?
Or to define an TFM_IDENTITY_KEY and then set that inside tfm_boards for each of the boards that use this?

Reply:
I added CONFIG_IDENTITY_KEY_TFM and use this here now. Can you check the newest code and see if this part makes sense to you?

Reply:
Seems okay, though the others should take a look. Now thinking, would it be possible to apply this logic to the already existing `CONFIG_IDENTITY_KEY`? Using `if TRUSTED_EXECUTION_*`... Not sure at all. Might or might not make sense.

Reply:
I think that using the CONFIG_IDENTITY_KEY_TFM at this case makes sense to me. The CONFIG_IDENTITY_KEY and CONFIG_IDENTITY_KEY_TFM have conflicting requirements since they target different builds Zephyr/TF-M. If we keep one symbol for these two we will have to provide two implementation of the library which doesn't seem better than what we have right now I think.